packages/client-core/src/components/Debug/index.tsx

```diff
@@ -105,18 +105,18 @@ const convertSystemTypeToDesiredType = (system: System): DesiredType => {
   return desired
 }
 
-export const Debug = ({ showingStateRef }) => {
-  useHookstate(getMutableState(EngineState).frameTime).value
+export const Debug = ({ showingStateRef }: { showingStateRef: React.MutableRefObject<boolean> }) => {
   const rendererState = useHookstate(getMutableState(RendererState))
   const engineState = useHookstate(getMutableState(EngineState))
+
+  engineState.frameTime.value // make Engine.instance data reactive in the render tree
```

Comment:
calling this does nothing and is unnecessary

Reply:
This is necessary for re-rendering the JSONTree's when anything changes. For example, if a new entity is added to the scene, the `namedEntities` data might not show it.